.github/workflows/diagram-cache.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: update diagram cache
+
+on: push
+
+jobs:
+    update-diagram-cache:
+      runs-on: ubuntu-latest
+      container:
+        image: ghcr.io/dsd-dbs/capella-dockerimages/capella/base:6.0.0-selected-dropins-main
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v3
+        - name: Install capella mbse
+          run:
+            pip install "git+https://github.com/DSD-DBS/py-capellambse.git@$CAPELLAMBSE_REVISION"
+        - name: Create diagram cache
+          run: |
+            mkdir diagram_cache
+            xvfb-run python <<EOF
+            import json
+            import os
+
+            import capellambse
+
+            ignore_duplicate_uuids = os.getenv(
+                "IGNORE_DUPLICATE_UUIDS_AND_VOID_ALL_WARRANTIES", ""
+            ).lower() in ("true", "1", "t")
+
+            resources = json.loads(os.getenv("RESOURCES", "null"))
+
+            print(f"capellambse.__version__: {capellambse.__version__}")
+            model = capellambse.MelodyModel(
+                os.environ["ENTRYPOINT"],
+                diagram_cache="diagram_cache",
+                ignore_duplicate_uuids_and_void_all_warranties=ignore_duplicate_uuids,
+                resources=resources,
+            )
+            model.update_diagram_cache(capella_cli="/opt/capella/capella", create_index=True)
+            EOF
```

Comment:
capellambse has a new CLI for the diagram cache: https://github.com/DSD-DBS/py-capellambse/blob/master/capellambse/diagram_cache.py

We should make use of it.

You can call it via: `python -m capellambse.diagram_cache ...`

Reply:
done